### DISCUSSION TITLE
Welcome to CKAN Discussions

### DISCUSSION THREAD

**Discussion by amercader** [ckan/ckan#5978] [General]
![Logo_263x86](https://user-images.githubusercontent.com/200230/112146805-49abe480-8bdc-11eb-9717-625c7d8bc64d.png)

## Welcome to CKAN Discussions
 
We’re using Discussions as a place to connect with other members of our community. 

Please choose the right category for the discussion you want to start!

  * Ask questions you’re wondering about.
  * Share ideas.
  * Ask for support with CKAN related issues
  * Introduce yourself and present how you use CKAN!

If you want to discuss a potential **security vulnerability** please follow this [link](https://github.com/ckan/ckan/security/policy).

Always remember to follow our [Code of Conduct](https://github.com/ckan/ckan/blob/master/CODE_OF_CONDUCT.md) when engaging in a discussion.

**Comment by jqnatividad** [comment]
Awesome! Just entered a "meta-idea" (#5991) 😃 

Can the [issue template](https://github.com/ckan/ckan/issues/new/choose) entry pointing to the Ideas repo also point here?
